index.html

```diff
@@ -38,5 +38,26 @@ <h2 class="text-container" id="sdsection">
       <img src="arrow-button.svg" class="arrow-button"><a href="#"></a></img>
       </div>
     </section>
+    <section class="works">
+      <h1 class="recent-works">My Recent Works</h1>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories Gain+Glory</h2>
```

Comment:
- [ ] Please style this header to look like the Figma template :+1: 
- [ ] Kindly make sure to do the same for all the other cards that will be added to this section.
- Your design
![image](https://user-images.githubusercontent.com/102699958/212462383-4f8956ae-f153-42ec-864e-429c242c2af2.png)
- The Figma design
![image](https://user-images.githubusercontent.com/102699958/212462406-ec7c1c74-5044-4b1c-a2f2-f0108f4e0f9f.png)